spring-boot/src/test/java/org/dcsa/conformance/springboot/ConformanceBasicAPITest.java

```diff
@@ -0,0 +1,117 @@
+package org.dcsa.conformance.springboot;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * Basic validation on API endpoints: testing content type, status code, and basic content.
+ * Does not run the actual conformance tests, because Tomcat is not started, so no actual endpoints are reachable by a httpClient.
```

Comment:
The tests does not need Tomcat to test hitting the API. It only skips the HTTP layer, which is not necessary to just hit it. So, basically we hit & test the front API layer. It does tests the API entry points for real. 

What the tests does is that it basically hits each different URL we're offering. It checks the: initial response, status code, content type, which is not checked anywhere else. The only thing we don't test, is the real starting of scenario/sandbox tests, which is fine, because we do that in an other test class.